addons/website_event/views/event_templates_page.xml

```diff
@@ -6,65 +6,77 @@
     <t t-call="website.layout">
         <!-- Options -->
         <t t-set="opt_events_list_categories" t-value="is_view_active('website_event.opt_events_list_categories')"/>
-        <div id="wrap" t-attf-class="o_wevent_event js_event d-flex flex-column #{'o_wevent_hide_sponsors' if hide_sponsors else ''}">
-            <t t-if="not event.menu_id">
-                <nav class="navbar navbar-light border-top shadow-sm d-print-none">
-                    <div class="container align-items-baseline justify-content-start">
-                        <a href="/event" class="navbar-brand h4 my-0 me-0 me-md-4">
-                            <i class="fa fa-long-arrow-left me-2"/>
-                            <span>All Events</span>
-                        </a>
-                        <ul class="navbar-nav flex-row ms-md-auto ms-0">
-                            <li t-if="opt_events_list_categories" class="nav-item me-3">
-                                <a t-attf-href="/event?type=#{event.event_type_id.id}" t-if="event.event_type_id" class="nav-link">
-                                    <i class="fa fa-folder-open me-2"/><span t-field="event.event_type_id"/>
-                                </a>
-                            </li>
-                            <li t-if="event.country_id" class="nav-item me-3">
-                                <a t-attf-href="/event?country=#{event.country_id.id}" class="nav-link">
-                                    <i class="fa fa-map-marker me-2"/><span t-field="event.country_id"/>
-                                </a>
-                            </li>
-                        </ul>
-                        <div class="d-flex align-items-centerflex-wrap ps-sm-3 pe-0">
-                            <t t-call="website_event.events_search_box_input">
-                                <t t-set="_classes" t-valuef="ms-auto"/>
-                            </t>
-                        </div>
-                    </div>
-                </nav>
-            </t>
-            <t t-else="">
-                <nav class="navbar navbar-light border-top shadow-sm navbar-expand-md">
+        <div id="wrap" t-attf-class="o_wevent_event js_event d-flex flex-column h-100 #{'o_wevent_hide_sponsors' if hide_sponsors else ''}">
+            <t t-if="not hide_submenu" t-call="website_event.navbar"/>
+            <t t-out="0"/>
+            <t t-set="editor_sub_message">Following content will appear on all events.</t>
+            <div class="oe_structure oe_empty" id="oe_structure_website_event_layout_1" t-att-data-editor-sub-message="editor_sub_message"/>
+        </div>
+    </t>
+</template>
+
+<template id="navbar" name="Event Navbar">
+    <div t-attf-class="container #{'my-3' if not event.menu_id else 'mt-3'}">
```

Comment:
Look s a bit too "crowded"... I'd suggest to use `my-3` everytime.
Beside making breath the layout, it would also be more consistent with the rest (notably the index page navbar).

| _ |
|--------|
| **This PR** |
| ![image](https://github.com/odoo/odoo/assets/8558351/80d7e861-aade-48bf-ad1a-1e27227891f5) |
| **Suggestion** |
| ![image](https://github.com/odoo/odoo/assets/8558351/dd04a108-519d-4eec-8560-ba42b7ef2677) |